baybe/acquisition/acqfs.py

```diff
@@ -30,12 +36,48 @@ class ExpectedImprovement(AcquisitionFunction):
 
 
 @define(frozen=True)
-class qProbabilityOfImprovement(AcquisitionFunction):
-    """Monte Carlo based probability of improvement."""
+class qExpectedImprovement(AcquisitionFunction):
+    """Monte Carlo based expected improvement."""
+
+    _abbreviation: ClassVar[str] = "qEI"
+
+
+@define(frozen=True)
+class qNoisyExpectedImprovement(AcquisitionFunction):
+    """Monte Carlo based noisy expected improvement."""
+
+    _abbreviation: ClassVar[str] = "qNEI"
+
+    prune_baseline: bool = field(default=True)
```

Comment:
```suggestion
    prune_baseline: bool = field(default=True, validator=instance_of(bool))
```

Reply:
I think this needs to be decided case by base (and yes, there are probably several still missing in the code). Here, it makes sense to have it, because I wouldn't want to put a `bool` converter, for example (you don't want to convert someone's `3` that has been mistakenly given into a `True`). However, if you have a field where we already have a `float` converter, there is no need to have an additional `instance_of(float)` validator. You know what I mean?